packages/turf-points-within-polygon/index.d.ts

```diff
@@ -11,9 +12,10 @@ import {
  * http://turfjs.org/docs/#pointswithinpolygon
  */
 export default function pointsWithinPolygon<
+  F extends Point | MultiPoint,
   G extends Polygon | MultiPolygon,
   P = Properties
 >(
-  points: Feature<Point, P> | FeatureCollection<Point, P>,
+  points: Feature<F, P> | FeatureCollection<F, P>,
   polygons: Feature<G> | FeatureCollection<G> | G
-): FeatureCollection<Point, P>;
+): FeatureCollection<F, P>;
```

Comment:
This makes the point input geometry generic as well, so that the return geometry will match what the user passes in.

Reply:
Yeah this is great.